source/making_payments/index.html.md.erb

```diff
@@ -77,37 +100,36 @@ datastore mapped to the payment ID just after you create a payment.
  * not paid because of a technical error
 
 Your service should use the API to check the payment status when the user
-reaches the return URL, and provide an appropriate response based on the final
+reaches the `return_url`, and provide an appropriate response based on the final
 status of the payment attempt.
 
 ## When a user does not complete their payment journey
 
-The user may close their browser or lose internet connection in the middle of
-the payment flow on GOV.UK Pay. These users will not be redirected back to
-your service.
-
-You can still check on the status of these payments by making a GET request
-using the Location Header or Self Link, the same way you would if they were
-redirected, but just after a set time (eg, two hours).
+Users may close their browser or lose internet connection in the middle of
+their payment journey on GOV.UK Pay. In this event, users will not be
+redirected back to your service.
 
->Note: GOV.UK Pay will expire incomplete payments after 90 minutes, but you should expect an occasional success or failure if the user experienced problems right at the moment of the redirect.
+You can still [check the status of these
+payments](/payment_flow/#check-the-status-of-a-payment), but GOV.UK Pay will
+expire incomplete payments after 90 minutes so you should only check after
+this amount of time.
 
-If a user does not have enough funds in their account to make a payment, the
-current GOV.UK Pay frontend will not let them try again with separate card
-details. They will be redirected to your service but you can offer them the option to try again. 
-That option would require your service to start a new payment. 
+If a [payment fails](/payment_flow/#payment-failure), GOV.UK Pay will not
+allow the user to attempt the payment again with alternative card details. In
+this situation the user would need to return to the service to retry the
+payment.
```

Comment:
Maybe this could be phrased in a way that focuses more on what the service should do to handle these cases. The second sentence could suggest that if the payment fails, the service should offer a way for the user to try again (which creates a new payment).

Reply:
Tried to make clearer by adding an extra sentence "Your service should account for this".